core/shared/src/main/scala/zio/prelude/ZValidation.scala

```diff
@@ -374,6 +374,7 @@ sealed trait ZValidation[+W, +E, +A] { self =>
 }
 
 object ZValidation extends LowPriorityValidationImplicits {
+  private val unitFailure: Validation[Unit, Nothing] = fail(())
```

Comment:
probably doesn't make sense to make it `lazy`, right?

Reply:
It doesn't indeed. It's had some synchronisation overhead